### PR TITLE
Rename `BTUIKPaymentOptionType` to `BTDropInPaymentMethodType`

### PR DESCRIPTION
**Pull request by sestevens**
### Summary of changes

Public API changes:
 - Rename `BTUIKPaymentOptionType` to `BTDropInPaymentMethodType`
 - Rename the `paymentOptionType` property on `BTDropInResult` to `paymentMethodType`

Internal changes:
 - Rename variables and methods containing `paymentOptionType` to `paymentMethodType`

 ### Checklist

 - [x] Added a changelog entry

- @sestevens 
- @scannillo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,15 +85,15 @@ func showDropIn(clientTokenOrTokenizationKey: String) {
 
 ### Apple Pay + Drop-In
 
-Apple Pay is enabled by default in Drop-In. Unless you opt out, by setting `showApplePayPaymentOption = false`, Drop-In will show Apple Pay as a payment option as long as it is enabled in the control panel. Below is an example of hiding the Apple Pay button if the device can't make Apple Pay payments using certain card networks:
+Apple Pay is enabled by default in Drop-In. Unless you opt out, by setting `applePayDisabled = true`, Drop-In will show Apple Pay as a payment option as long as it is enabled in the control panel. Below is an example of hiding the Apple Pay button if the device can't make Apple Pay payments using certain card networks:
```
Comment (sestevens) [inline README.md:88]: This isn't really related to the other changes in this PR -- I just happened to notice that this variable name was wrong.

**Review by sshropshire — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,8 @@
 * Fix iOS 13 UI bug where "Next" button overlapped with card number text field
 * Card form
   * Add validation to cardholder name field to prevent users from accidentally inputting credit card numbers
+* BTDropInResult
+  * Rename `paymentOptionType` property to `paymentMethodType`; rename `BTUIKPaymentOptionType` enum to `BTDropInPaymentMethodType`
```
Comment (scannillo) [inline CHANGELOG.md:13]: take it or leave it comment:

```suggestion
  * Rename `BTUIKPaymentOptionType` enum to `BTDropInPaymentMethodType`
  * Rename `paymentOptionType` property to `paymentMethodType`
```

File: V9_MIGRATION.md [hunk 3]
```diff
@@ -66,3 +66,5 @@ dropInRequest.uiCustomization = uiCustomization
 ## Drop-in Result
 
 The `isCancelled` property on `BTDropInResult` has been changed to `isCanceled`.
+
+The `paymentOptionType` property on `BTDropInResult` has been changed to `paymentMethodType`.
```
Comment (scannillo) [inline V9_MIGRATION.md:70]: Should we call out that the enum type name also changed?

**Review by scannillo — APPROVED** [review]

File: V9_MIGRATION.md (hunk 3 above)
Comment (sestevens) [reply V9_MIGRATION.md:70]:
```suggestion
The `paymentOptionType` property on `BTDropInResult` has been changed to `paymentMethodType`. The type of this enum has changed from `BTUIKPaymentOptionType` to `BTDropInPaymentMethodType`.
```

**Pull request merged**